themes/powerline-naked/powerline-naked.base.bash

```diff
@@ -25,9 +26,9 @@ function __powerline_left_segment {
 
   LEFT_PROMPT+="$(set_color ${params[1]} -)${pad_before_segment}${params[0]}${normal}"
   LAST_SEGMENT_COLOR=${params[1]}
-  (( SEGMENTS_AT_LEFT += 1 ))
+  ((SEGMENTS_AT_LEFT += 1))
 }
 
-function __powerline_left_last_segment_padding {
+function __powerline_left_last_segment_padding() {
```

Comment:
You can revert all the changes to this file.   We don't want to go down a rabbit hole of mixing formatting changes with new features.

Reply:
sublime automatically adds this for me :-)
I have removed it

Reply:
Ahh the curse of auto-formatting :)

I forgot to leave a note on the other changes in the file, but the same applies there.